src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1044,18 +1044,19 @@ private void OnAsyncTaskStateChanged(DynamoScheduler sender, TaskStateChangedEve
 
                         if (Logging.Analytics.ReportingAnalytics)
                         {
-                            var modifiedNodes = "";
                             if (updateTask.ModifiedNodes != null && updateTask.ModifiedNodes.Any())
                             {
-                                modifiedNodes = updateTask.ModifiedNodes
-                                    .Select(n => n.GetOriginalName())
-                                    .Aggregate((x, y) => string.Format("{0}, {1}", x, y));
+                                // Send analytics for each of modified nodes so they are counted individually
+                                foreach (var node in updateTask.ModifiedNodes)
+                                {
+                                    // Starting from Dynamo 2.11.0, tracking node execution as generic event so
+                                    // it is distinguished with the previous aggregated performance event
+                                    Dynamo.Logging.Analytics.TrackEvent(
+                                        Actions.Run, 
+                                        Categories.NodeOperations,
+                                        node.GetOriginalName());
+                                }
```

Comment:
I see this is not a timed event anymore, which makes sense to me given we cannot separate the individual execution time for each node. Just checking if this was an intended change since it wasn't announced in the description @QilongTang

Reply:
Thanks，yes it is intended and let me update description